src/dotnet/commands/dotnet-help/HelpCommandParser.cs

```diff
@@ -0,0 +1,25 @@
+﻿using Microsoft.DotNet.Cli.CommandLine;
+
+namespace Microsoft.DotNet.Tools.Help
+{
+    internal static class HelpCommandParser
+    {
+        public static Command Help()
+        {
+            return Create.Command(
+                "help",
+                LocalizableStrings.AppFullName,
+                Accept.ZeroOrOneArgument()
+                    .With(
+                        LocalizableStrings.CommandArgumentDescription,
+                        LocalizableStrings.CommandArgumentName),
+                Create.Option(
+                    "-h|--help|-?|/?",
+                    "Show help information",
```

Comment:
This should be moved to LocalizableStrings.cs. We are going to be converting those to resx + code-gen very soon to get CLI loc done and we shouldn't have any hard-coded strings which wouldn't be picked up by that conversion.